src/components/PeopleFilters.tsx

```diff
@@ -27,66 +98,87 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
+            <SearchLink
+              params={handleCenturiesChange('16')}
+              className={classNames(
+                'button',
+                'mr-1',
+                { 'is-info': centuries.includes('16') },
+              )}
               data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
             >
               16
-            </a>
-
-            <a
+            </SearchLink>
+            <SearchLink
+              params={handleCenturiesChange('17')}
+              className={classNames(
+                'button',
+                'mr-1',
+                { 'is-info': centuries.includes('17') },
+              )}
               data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
             >
               17
-            </a>
-
-            <a
+            </SearchLink>
+            <SearchLink
+              params={handleCenturiesChange('18')}
+              className={classNames(
+                'button',
+                'mr-1',
+                { 'is-info': centuries.includes('18') },
+              )}
               data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
             >
               18
-            </a>
+            </SearchLink>
+            <SearchLink
+              params={handleCenturiesChange('19')}
```

Comment:
Please, create an array for centuries and iterate over it to not repeat the same logic